### PR TITLE
[test-framework] Support helm values containing spaces

### PR DESCRIPTION
**Pull request by lei-tang**
This PR is for the issue: https://github.com/istio/istio/issues/13126.

### CODE REVIEW

File: pkg/test/helm/helm.go [hunk 1]
```diff
@@ -39,19 +41,30 @@ func Init(homeDir string, clientOnly bool) error {
...
-	}
-
 	valuesFileString := ""
 	if valuesFile != "" {
 		valuesFileString = fmt.Sprintf("--values %s", valuesFile)
 	}
 
-	out, err := shell.Execute("helm --home %s template %s --name %s --namespace %s %s %s",
-		homeDir, template, name, namespace, valuesFileString, valuesString)
+	str := fmt.Sprintf("helm --home %s template %s --name %s --namespace %s %s",
```
Comment (howardjohn) [inline pkg/test/helm/helm.go]: If you just make this an array to begin with it will he much simpler than creating a string and splitting it

Comment (lei-tang) [reply pkg/test/helm/helm.go]: Done.

File: pkg/test/helm/helm.go [hunk 2]
```diff
@@ -39,19 +39,21 @@ func Init(homeDir string, clientOnly bool) error {
...
-		homeDir, template, name, namespace, valuesFileString, valuesString)
+	// Override the values in the helm value file.
+	for k, v := range values {
+		if k == "" {
+			continue
+		}
+		p = append(p, "--set")
+		p = append(p, fmt.Sprintf("%s=%s", k, v))
+	}
+	out, err := shell.ExecuteArgs(nil, "helm", p[1:]...)
```
Comment (howardjohn) [inline pkg/test/helm/helm.go]: nit: no need to add `"helm"` as an arg than ignore it with `[1:]`

**Comment by lei-tang** [comment]
https://github.com/istio/istio/issues/13439

**Pull request merged**